src/ChinookSnippets/Chinook.DynamicMvvm.Combined/DynamicCommandFromActionWithParameter.snippet

```diff
@@ -0,0 +1,52 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+	<CodeSnippet Format="1.0.0">
+		<Header>
+			<Title>DynamicCommand with Parameter from Action</Title>
+			<Author>nventive</Author>
+			<Description>
+				Code snippet to implement a IDynamicCommand in a class implementing IViewModel.
+				You can also use "ckcmdap" to go faster.
+			</Description>
+			<HelpUrl></HelpUrl>
+			<SnippetTypes>
+				<SnippetType>Expansion</SnippetType>
+			</SnippetTypes>
+			<Keywords>
+				<Keyword>Chinook DynamicMvvm DynamicCommand Command Code Snippet</Keyword>
+			</Keywords>
+			<Shortcut>ckcmd</Shortcut>
+		</Header>
+		<Snippet>
+			<References>
+				<Reference>
+					<Assembly>Chinook.DynamicMvvm.Abstractions</Assembly>
+				</Reference>
+			</References>
+			<Imports>
+				<Import>
+					<Namespace>Chinook.DynamicMvvm</Namespace>
+				</Import>
+			</Imports>
+			<Declarations>
+				<Literal Editable="true">
+					<ID>CommandName</ID>
+					<Default>MyCommand</Default>
+				</Literal>
+				<Literal Editable="true">
+					<ID>PropertyType</ID>
```

Comment:
This should probably be name `ParameterType` since it's the parameter's type rather than the property's.